packages/fxa-settings/src/components/FormPasswordWithBalloons/index.tsx

```diff
@@ -120,6 +120,12 @@ export const FormPasswordWithBalloons = ({
     };
   }, []);
 
+  // TODO in FXA-7482, review our password requirements and best way to display them
+  // For now, this most closely matches parity to Backbone for a space-only password
+  const isTooShort =
+    errors.newPassword?.types?.length ||
+    (getValues('newPassword') && getValues('newPassword').trim() === '');
```

Comment:
Can we trim before validation?  Or does the validation not execute if the string is an empty string?

Reply:
If we trim before the validation, we disallow a space at the beginning or end of password inputs. That's currently allowed in Backbone, so I allowed it here.

Reply:
Yeah good thinking. If we throw in a `.trim()` before validation we'd disallow a space at the beginning or end but I changed `validate.length` to check for `value.length > 7 && value.trim() !== ''` instead of just the length there which I think is what you mean, and that works fine and is a smaller diff.